app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -14,6 +14,7 @@ class ConversionHostsController < BaseController
     #   * vmware_vddk_package_url
     #   * vmware_ssh_private_key
     #   * conversion_host_ssh_private_key
+    #   * openstack_tls_ca_certs
```

Comment:
Is this related or was it just missing before?

Reply:
It's not related, just thought I would add it while I was here. :)